pyramid_zipkin/tween.py

```diff
@@ -8,6 +9,72 @@
 from pyramid_zipkin.request_helper import get_binary_annotations
 
 
+ZipkinSettings = namedtuple('ZipkinSettings', [
+    'zipkin_attrs',
+    'transport_handler',
+    'service_name',
+    'span_name',
+    'add_logging_annotation',
+    'report_root_timestamp',
+])
+
+
+def _get_settings_from_request(request):
+    """Extracts Zipkin attributes and configuration from request attributes.
+    See the `zipkin_span` context in py-zipkin for more detaied information on
+    all the settings.
+
+    Here are the supported Pyramid registry settings:
+
+    zipkin.create_zipkin_attr: allows the service to override the creation of
+        Zipkin attributes. For example, if you want to deterministically
+        calculate trace ID from some service-specific attributes.
+    zipkin.transport_handler: how py-zipkin will log the spans it generates.
+    zipkin.stream_name: an additional parameter to be used as the first arg
+        to the transport_handler function. A good example is a Kafka topic.
+    zipkin.add_logging_annotation: if true, the outermost span in this service
+        will have an annotation set when py-zipkin begins its logging.
+    zipkin.report_root_timestamp: if true, the outermost span in this service
```

Comment:
hmm... you know you are a root span, because you have extracted no trace ids from headers. I think that's an unambiguous signal that you are not going to have a corresponding client span.

Why would we put an option since the impl knows this for sure?